src/util.rs

```diff
@@ -121,6 +123,61 @@ pub async fn hash_all(files: &[PathBuf], pool: &tokio::runtime::Handle) -> Resul
     Ok(hashes)
 }
 
+/// Calculate the digest of each Unix archive file in `files` on background threads in
+/// `pool`.
```

Comment:
This comment might add confusion more than it clarifies - `hash_all_archives()` will also operate on Windows static libraries as well.

I can see why you've specified this as "Unix archive", since that's how it's specified in the [`ar` crate](https://crates.io/crates/ar). However, digging into the project more, we can see [that people are using it with non-Unix OSes](https://github.com/mdsteele/rust-ar/issues/19).

```suggestion
/// Calculate the digest of each static library archive in `files` on background threads in
/// `pool`.
```